File: test_samples/valid/no_complex_boolean_condition.ts
function shouldRender(contact: Contact): boolean {
  if (contact) {
    const isInactive = !contact.isActive();
    const isFamilyOrFriends = contact.inGroup(FAMILY) || contact.inGroup(FRIENDS);
    if (isInactive && isFamilyOrFriends) {
      return true;
    }
  }
  return false;
}

File: test_samples/valid/no_complex_boolean_condition.py
def should_render(contact: Contact) -> bool:
    if contact:
        is_inactive = not contact.is_active()
        is_family_or_friends = contact.in_group(FAMILY) or contact.in_group(FRIENDS)
        if is_inactive and is_family_or_friends:
            return True
    return False
